recipes/dev/lora_finetune_fsdp2.py

```diff
@@ -230,6 +231,10 @@ def setup(self, cfg: DictConfig) -> None:
         )
 
         self._loss_fn = config.instantiate(cfg.loss)
+        if self._model_compile:
```

Comment:
nit: should we rename "model_compile" to just "compile", since this command will control all loss compile and flexattention compile?

Reply:
not a priority, i am fine to keep as it for now, and maybe add a todo to rename

Reply:
Yeah I think our naming here is not great. Let's do in a follow-up; rather than add a todo in the code I may just create an issue.

Reply:
[#](https://github.com/pytorch/torchtune/issues/1289)

Reply:
#1426